src/tools/PrizeTierController/Actions.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button, ButtonTheme, ButtonSize } from '@pooltogether/react-components'
+import { checkForPrizeCompatibility } from '@prizeTierController/utils/checkForPrizeCompatibility'
+import {
+  allCombinedPrizeTiersAtom,
+  isSavePrizeTiersModalOpenAtom,
+  prizeTierEditsAtom
+} from '@prizeTierController/atoms'
+import classNames from 'classnames'
+import { useAtom } from 'jotai'
+import { useResetAtom, useUpdateAtom } from 'jotai/utils'
+import { useUsersAddress } from '@pooltogether/wallet-connection'
+
+export const Actions = (props: { className?: string }) => {
+  const { className } = props
+  const setIsOpen = useUpdateAtom(isSavePrizeTiersModalOpenAtom)
+  const resetForm = useResetAtom(prizeTierEditsAtom)
+  const [allPrizeTierEdits] = useAtom(prizeTierEditsAtom)
+  const [combinedPrizeTiers] = useAtom(allCombinedPrizeTiersAtom)
+  const usersAddress = useUsersAddress()
+
+  const editedPools: { chainId: string; address: string }[] = []
```

Comment:
Any code at the top level of the component will run on each render. Since we have some high level components that refetch data on intervals, components can render a lot. Most of the time you won't want to do any sort of looping through data here since it can happen **a lot**. You'll usually want to put it inside of a `useMemo` or `useEffect` hook to react and only do the looping when the data you're looking at changes.

This is a unique case where it's probably fine to leave it. Just something to look out for!

The react dev tool helps visualize this well.
<img width="470" alt="Screen Shot 2022-11-21 at 3 25 01 PM" src="https://user-images.githubusercontent.com/13701258/203151969-efc89398-d29e-4e69-a90b-2d5bce596bb4.png">